examples/tokenizer_training.ipynb

```diff
@@ -23,7 +23,7 @@
    "outputs": [],
    "source": [
     "#! pip install datasets\n",
-    "#! pip install git+https://github.com/huggingface/transformers[sentencepiece]"
+    "#! pip install git+https://github.com/huggingface/transformers#egg=transformers[sentencepiece]"
```

Comment:
I propose this change because with the initial command I was getting this error: 
```
ERROR: Command errored out with exit status 128: git clone -q 'https://github.com/huggingface/transformers[sentencepiece]' /tmp/pip-req-build-zdg7pzs4 Check the logs for full command output.
```